TourMate/TourMate/Views/Trip/TripView.swift

```diff
@@ -8,67 +8,93 @@
 import SwiftUI
 
 struct TripView: View {
+    @Environment(\.dismiss) var dismiss
+
     @StateObject var plansViewModel: PlansViewModel
+    @StateObject var viewModel: TripViewModel
```

Comment:
Maybe can rename to `tripViewModel` cause we have 2 view models here

Reply:
Ok I will try moving PlansViewModel to PlansListView